resources/views/reports/custom.blade.php

```diff
@@ -283,6 +283,16 @@
               </div>
             </div>
 
+          <!-- Checkout Date -->
+          <div class="form-group purchase-range">
```

Comment:
Nothing really needs to change here but I noticed the `purchase-range` class and was going to call it out but noticed that two other divs already use it. Changing it to be `checkout-range` would require another `.datepicker()` chunk in the `script` section at the bottom of the file. Or...I think jQuery can accept a selector like 
```javascript
$(['.purchase-range .input-daterange'], ['.created-range .input-daterange'], ['.checkout-range .input-daterange']).datepicker({
    clearBtn: true,
    todayHighlight: true,
    endDate: '0d',
    format: 'yyyy-mm-dd'
});
```

but again...it's not actually needed.

Reply:
I added it just to keep everything uniform.